git_machete/git_operations.py

```diff
@@ -318,9 +319,11 @@ def get_config_attr_or_none(self, key: str) -> Optional[str]:
         self.__ensure_config_loaded()
         return self.__config_cached.get(key.lower())
 
-    def get_boolean_config_attr(self, key: str) -> bool:
+    def get_boolean_config_attr(self, key: str, default_value: bool = True) -> bool:
```

Comment:
Pls remove the default value for this param completely. The longer I live, the more convinced I am that the default param values are a cancer

![image](https://user-images.githubusercontent.com/3383210/174269805-ebcaaa66-7dd2-4457-abd4-56794ff92be9.png)